recipes/interpret-core/meta.yaml

```diff
@@ -0,0 +1,88 @@
+{% set name = "interpret-core" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  git_url: https://github.com/interpretml/interpret.git
+  git_rev: "v{{ version }}"
+  git_depth: 1
+
+build:
+  skip: true  # [py>=39 or py2k or py<37]
+  skip: true  # [win or osx]
+  number: 0
+  script: cp -r shared python/interpret-core/symbolic/shared
+  script: "npm install --package-lock-only --ignore-scripts && npx force-resolutions"
+  script: cd python/interpret-core && {{ PYTHON }} setup.py build && {{ PYTHON }} setup.py install
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - gxx  # [linux]
+    - nodejs >=16.0,<17.0
+
+  host:
+    - pip
+    - python
+  
+  run:
+    - python
+    # Equiv. to required.
+    - scikit-learn >=0.18.1
+    - numpy >=1.11.1
+    - scipy >=0.18.1
+    - pandas >=0.19.2
+    - joblib >=0.11
+
+  run_constrained:
+    # Equiv. to required.
+    - scikit-learn >=0.18.1
+    - numpy >=1.11.1
+    - scipy >=0.18.1
+    - pandas >=0.19.2
+    - joblib >=0.11
+    # Equiv. to extras.
+    - ipython >=7.4.0
+    - ipykernel >=5.1.0
+    - plotly >=3.8.1
+    - psutil >=5.6.2
+    - lime >=0.1.1.33
+    - SALib >=1.3.3
+    - shap >=0.28.5
+    - skope-rules >=1.0.1    
+    - treeinterpreter >=0.2.2
+    - dill >=0.2.5
+    # Equiv. to Dash.
+    - dash >=1.0.0
+    - dash-cytoscape >=0.1.1
+    - gevent >=1.3.6
+    - requests >=2.19.0
```

Comment:
Please only include the required dependencies here. If you want to install extra dependencies, please do that via an additional output of the recipe that depends on more of them.

```suggestion
```